src/frontend/components/Felleskomponenter/Steg/Steg.tsx

```diff
@@ -165,10 +174,14 @@ const Steg: React.FC<ISteg> = ({ tittel, guide, skjema, gåVidereCallback, child
 
     const { tilbakeKnapp } = tekster().FELLES.navigasjon;
 
+    const dokumentasjonTekster = tekster().DOKUMENTASJON;
     const frittståendeOrdTekster = tekster().FELLES.frittståendeOrd;
 
     const formProgressStegOppsummeringTekst = `${plainTekst(frittståendeOrdTekster.steg)} ${hentNåværendeStegIndex()} ${plainTekst(frittståendeOrdTekster.av)} ${formProgressSteg.length}`;
 
+    const skalVedleggOppsummeringVises =
```

Comment:
Personlig syns jeg det er greit å trekke ut slik logikk ut i fra selve komponenten slik at det enkelt kan testes uten å blande inn UI. Litt usikker på om det er lagt opp til det i prosjektet, men det er ikke uvanlig å ha egne filer for logikk som dette. 😄